### PR TITLE
hypervisorCP for API1000/1200

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
Added Support for Hypervisor Cluster Profile for API 1000 and API 1200

### Issues Resolved
N/A

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ y ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ y ] New endpoints supported are updated in the endpoints-support.md file.
- [ y ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: lib/oneview-sdk/resource/api1000/c7000/hypervisor_cluster_profile.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api800/c7000/hypervisor_cluster_profile'
+
+module OneviewSDK
+  module API1000
+    module C7000
+      # Hypervisor cluster profile resource implementation for API1000 Synergy
```
Comment (soodpr) [inline lib/oneview-sdk/resource/api1000/c7000/hypervisor_cluster_profile.rb:17]: It should be c7000

File: lib/oneview-sdk/resource/api1200/c7000/hypervisor_cluster_profile.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require_relative '../../api1000/c7000/hypervisor_cluster_profile'
+
+module OneviewSDK
+  module API1200
+    module C7000
+      # Hypervisor cluster profile resource implementation for API1200 Synergy
```
Comment (soodpr) [inline lib/oneview-sdk/resource/api1200/c7000/hypervisor_cluster_profile.rb:17]: replace Synergy with C7000

Comment (AsisBagga) [reply lib/oneview-sdk/resource/api1200/c7000/hypervisor_cluster_profile.rb]: Corrected.

File: lib/oneview-sdk/resource/api1000/c7000/hypervisor_cluster_profile.rb (hunk 1 above)
Comment (AsisBagga) [reply lib/oneview-sdk/resource/api1000/c7000/hypervisor_cluster_profile.rb]: Corrected.

**Pull request merged**